apps/machine/zynqmp_r5/platform_info.c

```diff
@@ -100,9 +105,11 @@ platform_create_proc(int proc_index, int rsc_index)
 
 	(void) proc_index;
 	rsc_table = get_resource_table(rsc_index, &rsc_size);
-
+#ifndef RPMSG_NO_IPI
 	/* Register IPI device */
 	(void)metal_register_generic_device(&ipi_device);
+#endif /* !RPMSG_NO_IPI */
```

Comment:
Would be nice here to propagate the error returned by metal_register_generic_device... ( in both cases)

Reply:
done